src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  font-family: Roboto, Arial, 'Helvetica Neue', Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should add a generic family name as a fallback. For example: font-family: Roboto, Arial, 'Helvetica Neue', Helvetica, sans-serif, sans-serif;